app/views/shared/_analytics.html.erb

```diff
@@ -0,0 +1,11 @@
+<% if ENV['GA_ANALYTICS_ID'] %>
```

Comment:
This will work on prod, but it didn't look like int had a GA_ANALYTICS_ID. Should it have? If not, I could just rename the variable on stg to make this work. When we remove the UA code and property after June, it might be good to keep the more generic variable name anyway.

Reply:
I actually changed the comment on the issue and thought we should have GA_ANALYTICS_ID on catalog-int. Can you rename the variable? After Universal Analytics is down we can update it to remove the UA ID from production. Make sense?